pkg/skaffold/runner/timings.go

```diff
@@ -54,6 +57,17 @@ func (w withTimings) Build(ctx context.Context, out io.Writer, tagger tag.Tagger
 	return bRes, err
 }
 
+func (w withTimings) Test(out io.Writer, builds []build.Artifact) error {
+	start := time.Now()
+	color.Default.Fprintln(out, "Starting test...")
+
+	err := w.Tester.Test(out, builds)
+	if err == nil {
```

Comment:
Reverse this logic

Reply:
```golang
if err != nil {
return errors.Wrap(...)
}

Fprintln
return nil
```

Reply:
I copied this logic from the other timing methods in this file so this would match. Do you want me to change them all?